### PR TITLE
feat: container log api

### PR DESCRIPTION
**Pull request by robot9706**
Add REST APIs for container logs.

### CODE REVIEW

**Review by polaroi8d — APPROVED** [review]
Please wait for further approval from @nandor-magyar for minor questions / requests.

File: web/crux/src/app/node/node.global-container.http.controller.ts
Comment (nandor-magyar) [reply web/crux/src/app/node/node.global-container.http.controller.ts:124]: It is fine, I am not sure if it is perfect here, but we have a `/log` endpoint elsewhere as well.

**Review by nandor-magyar — APPROVED** [review]
LGTM

**Pull request merged**
